### DISCUSSION TITLE
I've been thinking that the Oscar theme needs a few stylistic adjustments, and fixes...

### DISCUSSION THREAD

**Discussion by SecularSteve** [searx/searx#2984] [Ideas]
For example, the Categories boxes look a bit better (in my opinion), when they're "dashed" instead of "solid":

![1](https://user-images.githubusercontent.com/33793273/134882648-b030bc9e-fc0c-4c82-a77d-0f3f435bf82b.png)

And the Languages Box needs an "overflow" border with 3 dots spawning automatically when a cetrain pixel width limit is reached by the text:

![2](https://user-images.githubusercontent.com/33793273/134882690-655c3b6c-1774-4934-afe7-952810ffb31b.png)
![3](https://user-images.githubusercontent.com/33793273/134882694-39c9826a-5317-41dd-a7c8-fa73302545f5.png)

On top of that, it could be made that only if the mouse hovers over the box itself that it can slowly scroll to the left with the...

`<marquee behavior="scroll" direction="left"`

...command.

Like so, but only if it overflows:

![4](https://user-images.githubusercontent.com/33793273/134882741-e3bb9ecc-ca61-444e-98e2-e5864d0e4ece.gif)

**Comment by 0xhtml** [comment]
I implemented the first part of the overflow suggestion.
Take a look: #2985
I'm not sure if the marquee part is realistically possible though.
